src/vehicle.cpp

```diff
@@ -623,6 +624,21 @@ void vehicle::init_state( int init_veh_fuel, int init_veh_status )
     invalidate_mass();
 }
 
+void vehicle::activate_magical_follow()
+{
+    for( size_t e = 0; e < parts.size(); e++ ) {
+        const vehicle_part &vp = parts[ e ];
+        if( vp.info().fuel_type == fuel_type_mana ) {
+            parts[ e ].enabled = true;
```

Comment:
```suggestion
            vp.enabled = true;
```
And make `vp` non-const.